src/zope/container/_compat.py

```diff
@@ -0,0 +1,6 @@
+try:
+    # PY2
+    text_type = unicode
+except NameError:
+    # PY3
+    text_type = str
```

Comment:
Is this actually used more than once?

Reply:
I am not sure. But it is in the _compat module, so it will go away sooner or later.